docs/source/gbd2019_models/risk_effects/fpg/index.rst

```diff
@@ -0,0 +1,256 @@
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+======================
+Fasting Plasma Glucose 
+======================
+
+
+.. contents::
+   :local:
+   :depth: 2
+
+Risk Overview
+-------------
+
+Glucose is the primary energy source of the cells of the human body. Homeostasis of glucose metabolism is assessed by measuring plasma glucose levels either by 
+measuring fasting plasma glucose (FPG), performing an oral glucose tolerance test (OGTT), or analyzing glycated hemoglobin (HbA1c). These measurements can be used 
+to classify subjects as having normal glucose metabolism (FPG <100 mg/dL, OGTT <140 mg/dL, HbA1c <5.7%), impaired glucose metabolism (FPG 100 to <126 mg/dL, 
+OGTT 140 to 199 mg/dL, HbA1c 5.7 to 6.4%), or diabetes mellitus (FPG >=126 mg/dL, OGTT >=200 mg/dL, HbA1c >=6.5%). HbA1c is specific for diabetes but not very sensitive 
+and has greater utility to monitor diabetes control over 2 to 3 months.
+
+[Normal-FPG-Levels_effects]_
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+Risk exposure
++++++++++++++
+
+:ref:`See risk exposure documentation for FPG <2019_risk_exposure_fpg>`
+
+Relative risks
+++++++++++++++
+
+FPG is a risk factor for many causes which are listed in the table 
+below. For the CVD model, FPG will affect ischemic heart disease, 
+and ischemic stroke. There were no updates to GBD modeling of FPG for these causes. 
+
+RRs were reported per 1 mmol/L increase in FPG above the TMREL value 
+(4.8-5.4 mmol/L), calculated as in the equation below: 
+
+:math:`\text{RR(x)} = {\text{RR}_0}^{\max\left((x-\text{TMREL}), 0\right)}`
+
+Where RR(x) is the RR at exposure level x and RR\ :sub:`0`\  is the 
+increase in RR for each 1 mmol/L above the TMREL. GBD used 
+DisMod-MR 2.1 to pool effect sizes from included studies and generate 
+a dose-response curve for each of the outcomes associated with high SBP. 
+The tool enabled GBD to incorporate random effects across studies and 
+include data with different age ranges. RRs were used universally for 
+all countries and the meta-regression only helped to pool the three 
+major sources and produce RRs with uncertainty and covariance across 
+ages taking into account the uncertainty of the data points. 
+
+
+
+Theoretical minimum-risk exposure level
++++++++++++++++++++++++++++++++++++++++
+
+The theoretical minimum-risk exposure level (TMREL) for FPG is 4.8-5.4 mmol/L for those risk-outcome pairs where risk is assessed on a continuous basis. This was calculated by taking the person-year 
+weighted average of the levels of FPG that were associated with the lowest risk of mortality in the pooled analyses of prospective cohort studies. The TMREL is no diabetes for those outcomes where risk 
+is assessed on a categorical basis. The risk-outcome pairs are listed below, along with whether they are continuous or categorical.  
```

Comment:
Is this a range because it varies by age/sex? Or is the bottom end of the range a minimum value for FPG exposure? This wasn't clear to me on a quick read!